person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age || parent_permission == true
+
+    false
+  end
```

Comment:
- [Optional] Hi @usorfaitheloho, you can simply go like this:

```ruby
     def can_use_services? 
            of_age?() || @parent_permission
     end
```
The same logic can also be applied to the private method `of_age`, since by default the method `of_age?` will return true otherwise false.

Reply:
Thanks for your suggestion, I have implemented the changes.